herbert_core/herbert_init.m

```diff
@@ -82,7 +82,7 @@ function addgenpath_message (varargin)
 % T.G.Perring
 
 string=fullfile(varargin{:},'');    % '' needed to circumvent bug in fullfile if only one argument, Matlab 2008b (& maybe earlier)
-if exist(string,'dir')==7
+if exist(string, 'dir')
```

Comment:
A comment would be useful here noting that this call to exist() is required because it's before the is_folder() method is added to the path etc... That would stop an overzealous future developer spotting "the one exist that you forgot to change" and 'fixing' it

Can the variable name `string` be changed while this file is being updated? string is a type name not a variable name